README.md

```diff
@@ -15,8 +15,10 @@ see the **[elasticsearch-rails](https://github.com/elasticsearch/elasticsearch-r
 The Elasticsearch client is compatible with Ruby 1.9 and higher.
 Other libraries in this repository might require a more recent Ruby version.
 
-The client's API is compatible with Elasticsearch's API versions from 0.90 till current,
-just use a release matching major version of Elasticsearch.
+The client's API is compatible with Elasticsearch's API versions from 0.90 till
```

Comment:
This seems ambiguous to me, its the first thing stated so reads like its compatible with `0.90` and up. 

Would mention in this order:

- clients are locked with major version
- pick a client with the same major version
- this client has had releases since 0.90
- refer to Elastic EOL policy to understand which versions are actively supported/tests

Reply:
@Mpdreamz Thanks for the review, I made these changes and I'm adding the EOL policy link to the Rails client too.